### ISSUE TITLE
ATL.width and ATL.height enhancement

### ISSUE THREAD

**Issue by MotoMoto1234** [kandashi/Active-Token-Lighting#136]
**Is your feature request related to a problem? Please describe.**
No not really

**Describe the solution you'd like**
I would like to be able to utilize the ADD mode for ATL.width and ATL.height.  The use case is for when a Rune Knight in 5e enlarges with Giant's Might and then drinks an Enlarge/reduce potion.  

**Describe alternatives you've considered**
Only way I know how to account for this if not using ADD mode is to directly edit the tokens

**Additional context**
Basically stacking two sources of token size changes.  I'd use ATL.width | ADD | +1

**Comment by kaelad02** [comment]
Changing them with Add works for me. I created an active effect like:

```
ATL.height | Add | 1
ATL.width  | Add | 1
```

and it worked as expected. Since those properties are numbers and not die rolls, you don't need to add the `+` in front of them but it did happen to work with it.

**Comment by kaelad02** [comment]
I just tried with a second effect using the same keys, jut to be sure, and they worked too. Enabling one would make the token 2x2, the second 3x3. Disabling the effects, even in a different order, worked as expected.

**Comment by MotoMoto1234** [comment]
ok, I will try again, I probably had a module conflict.

**Comment by kaelad02** [comment]
Closing since there's no update. Can always reopen if the issue was with ATE.
